### ISSUE TITLE
Hardware guidance to run a delta node

### ISSUE THREAD

**Issue by anjor** [application-research/delta#114]
Hardware recommendations for running a delta node, so that clients who want to run their own delta node know how much infra to provision.

**Comment by alvin-reyes** [comment]
Yes. Thank you for raising this. Delta is lightweight so the hardware requirement to run is not demanding.

I'll add it here and on our set up docs.

**Comment by alvin-reyes** [comment]
To run delta

- Processor: Any multi-core processor with a clock speed of at least 1.5 GHz. AMD based processors recommended (for commp).
- RAM: RAM requirement will vary based on usage but At least 1 GB of RAM to run and process small datasets.
- Hard Drive: Storage requirement will vary based on the dataset that Delta will process. At least 50 GB of storage space.
- Operating System: Tested on MacOS, Linux. Reportedly works as expected on Windows.
- Networking: A standard network interface card (NIC) with a speed of at least 100 Mbps. May vary depending on how large the data will be since online/e2e deals can use considerable amount of egress.

**Note that these requirements may vary depending on the specific needs of your application.**

**Comment by alvin-reyes** [comment]
https://github.com/application-research/delta/blob/main/docs/getting-started-run-delta.md

**Comment by anjor** [comment]
Awesome, thanks @alvin-reyes
